ui/desktop/app/styles/app.scss

```
@@ -467,6 +467,10 @@ $chevron-down: url("data:image/svg+xml;utf-8,<svg viewBox='0 0 24 24' fill='%231
     width: 100%;
     justify-content: space-between;
 
+    .copyable-content > pre {
+      white-space: pre-wrap;
+    }
```

I changed this to `line-height` and it fixed the line height issue while maintaining the format we want for objects. 

```suggestion
    .copyable-content > pre {
      line-height: normal;
    }
```